blockchain/for-companies/how-to-run-liberland-company.md

```diff
@@ -0,0 +1,132 @@
+Your company might operate under a legally binding contract, or you might want to sign contracts with clients or other parties.
+You can reference these contracts directly in your company registration if you so wish.
+
+[Browse existing Contracts](www.blockchain.liberland.org/home/contracts/overview)
+
+To create a contract, 
+
+TODO CONTRACT CREATION
+
+Note that contracts might require legal expertise to set it up properly. 
+Feel free to ask for help, there should be companies and professionals offering these services.
+
+We are working on a easy to use contract builder but it will take time to finish.
+
+After the contract is ready, copy the link of the contract send it to the parties and judges to sign. 
+The most common 'judge' is the official court of Liberland which you can contact here
+TODO ADD LIBERLAND COURT CONTACT
+
+### Handling payments
+
+While Liberland does not have any regulations on methods, currencies, or reporting of payments, 
+it is good for Liberland to do business on the Liberland chain, especially if you are engaging in legally binding contracts.
+
+Assets that are currently available on Liberland blockchain are LLM, LLD and stablecoins like LLEUR and USDT.
+Bitcoin and other cryptocurrencies are welcome, but they are of course on their own chains and not on Liberland blockchain.
+However, 'wrapped' versions are available through 'bridges', and you can do business in 'bitcoin stablecoins' on Liberland blockchain.
+
+LLD is the easiest, as it can be exchanged to USDT, BTC, other crypto or fiat like USD and EUR.
+This means that you can do business with LLD internationally, or with any other crypto. 
+
+If you so wish, you can issue an invoice to the client in USD, convert it into LLD, pay for services of another company with the LLD,
+or convert it into Bitcoin for long term investment, or of course, any other combination. 
+We are not here to tell you what to do with your money, we are just here to support you if you need help figuring out how to do it.
+
+TODO LINK TO EXCHANGING LLD
+
+### Fundraising, assets and the stock market
+
+You can issue stock of your company and list it for sale on the Liberland DEX stock market for fundraising, or any other purposes.
+
+There are no regulations in Liberland how to do it, but if you want to do it on chain, the proper way is to do it through an on-chain asset.
+
+This will cost 200 LLD.
+
+TODO interface for asset creation
+
+Once your asset has been created, its good practice to connect it to an shareholder contract. 
+As usual, there are no regulations about it in Liberland, but 
+this contract should give people the incentive to buy the stake in your company, like percentage of profits or voting rights 
+what is common with company stock worldwide.
+
+An example of a shareholder contract can be found here
+
+TODO shareholder contract
+
+You can now list it on the Liberland DEX stock market where you can set the price and people can buy and trade it
```

Comment:
please replace with: "You now have the option to list your asset on the Liberland Decentralized Exchange (DEX), where you can establish its price, allowing others to purchase and trade it."